psd-web/spec/features/report_product_spec.rb

```diff
@@ -317,14 +448,19 @@ def expect_case_activity_page_to_show_test_result(test)
     expect(item).to have_text(test[:details])
   end
 
+  def fill_in_coronavirus_page(answer)
+    choose answer ? "Yes, it is (or could be)" : "No, this is business as usual"
```

Comment:
Suggest scoping the the answer to a fieldset, to help make sure that the HTML is correct and improve readability of the spec:

```suggestion
    within_fieldset("Is this case related to the coronavirus outbreak?") do
        choose answer ? "Yes, it is (or could be)" : "No, this is business as usual"
    end
```